app/assets/stylesheets/ideas.scss

```diff
@@ -1,3 +1,10 @@
-// Place all the styles related to the Ideas controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+#deleted {
+  background: #ffe;
+  border: 1px solid #808080;
```

Comment:
Color literals like `#808080` should only be used in variable declarations; they should be referred to via variable everywhere else.